src/api/elastic.rs

```diff
@@ -221,10 +221,14 @@ pub struct ResponseHit {
 
 #[derive(Deserialize)]
 pub struct ResponseSource {
+    #[serde(default = "String::default")]
```

Comment:
If these strings are empty does that mean the user will see strange empty search results or are they filtered elsewhere before going back through the API?

Reply:
yeah, they will - this is how the error-case ends up being rendered with this change:

![image](https://user-images.githubusercontent.com/755354/186154906-b7ef833d-1c54-44ff-a82f-b5f1145de548.png)

I think this is ok - the client makes a best-effort of rending the results - and better than rumba erroring out, but I'm not sure.